cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/hvdc/TPnodeEquipments.java

```diff
@@ -86,70 +87,70 @@ private void computeThreeWindingsTransformer(CgmesModel cgmesModel, Adjacency ad
         CgmesTerminal t3 = cgmesModel.terminal(end3.getId(CgmesNames.TERMINAL));
 
         String id = end1.getId(CgmesNames.POWER_TRANSFORMER);
-        List<String> topologicalNodes = new ArrayList<>();
-        topologicalNodes.add(t1.topologicalNode());
-        topologicalNodes.add(t2.topologicalNode());
-        topologicalNodes.add(t3.topologicalNode());
-        if (isValidTransformer(adjacency, topologicalNodes)) {
-            addTransformer(adjacency, id, topologicalNodes, EquipmentType.TRANSFORMER);
+        List<String> nodes = new ArrayList<>();
+        nodes.add(CgmesDcConversion.getAcNode(cgmesModel, t1));
+        nodes.add(CgmesDcConversion.getAcNode(cgmesModel, t2));
+        nodes.add(CgmesDcConversion.getAcNode(cgmesModel, t3));
+        if (isValidTransformer(adjacency, nodes)) {
+            addTransformer(adjacency, id, nodes, EquipmentType.TRANSFORMER);
         }
     }
 
-    private void addEquipment(Adjacency adjacency, String id, String topologicalNodeId1, String topologicalNodeId2,
+    private void addEquipment(Adjacency adjacency, String id, String nodeId1, String nodeId2,
```

Comment:
Since we are not relying always on TP, I would suggest to rename this class as `NodeEquipment` instead of `TPnodeEquipments`. Also, consider removing the final "s" from "equipments" (word is uncountable and should not be used in plural).

Reply:
The inner static class `TPnodeEquipment`, that stores an identifier and type of a piece of equipment could be renamed `EquipmentReference` to avoid naming collision with the main class.

Reply:
Done